src/services/NotificationTypeService.js

```diff
@@ -0,0 +1,113 @@
+import { isNil, pick } from 'lodash';
+import { getListFilters } from '../utils/filter';
+import NotificationType from '../models/notificationType';
+import logger from '../utils/logger';
+import { ApiError } from '../utils/errors';
+
+const notificationTypesFilters = ['_id', 'version'];
```

Comment:
Add the name, description, and template_id fields to the list of filters.

Reply:
done